Server/API/RemoteControlController.cs

```diff
@@ -114,17 +118,17 @@ bool remoteControlStarted()
                 {
                     return StatusCode(408, "The remote control process failed to start in time on the remote device.");
                 }
```

Comment:
If we successfully start a remote control session here, we're not doing anything with it.  We drop below the if/else and return BadRequest.